xbmc/pvr/windows/GUIWindowPVRRecordings.cpp

```diff
@@ -99,6 +100,24 @@ bool CGUIWindowPVRRecordingsBase::OnAction(const CAction& action)
       return true;
     }
   }
+  else if (action.GetID() == ACTION_TOGGLE_WATCHED)
+  {
+    const std::shared_ptr<CFileItem> pItem = m_vecItems->Get(m_viewControl.GetSelectedItem());
+    if (!pItem || pItem->IsParentFolder())
+      return false;
+
+    bool bUnWatched;
```

Comment:
Although this cannot happen with current implementation, I would initialize this variable. Makes the code more robust in case of future functional changes.

Reply:
I've adjusted to give it a default of false